makerdao/governance-portal-v2
# Add hat badge to exec detail

### Link to Clubhouse story

https://app.shortcut.com/dux-makerdao/story/563/show-governing-proposal-status-on-exec-detail-page

### What does this PR do?

Adds "Governing Proposal" badge to exec detail page

### Steps for testing:

1. Go to governing proposal page
2. Observe badge ✨ 

### Screenshots (if relevant):

<img width="852" alt="Screen Shot 2021-11-03 at 2 15 04 PM" src="https://user-images.githubusercontent.com/5225766/140067993-1aed4525-6226-4591-a704-3f8bca7aff29.png">
<img width="372" alt="Screen Shot 2021-11-03 at 2 14 57 PM" src="https://user-images.githubusercontent.com/5225766/140067985-d7027ed7-10ed-45e8-9e1d-2d88e8261b9a.png">

### Add a GIF:
![](https://media.giphy.com/media/l0MYKY4LOLWTGiCSA/giphy.gif)
